### PR TITLE
Add very important data to editor dummy data

### PR DESCRIPTION
**Pull request by Aberdeener**

### CODE REVIEW

**Comment by Evidentsinger14** [comment]
pls don't merge, the infamous Ev1dent was not included.

**Comment by Turbotailz** [comment]
> pls don't merge, the infamous Ev1dent was not included.

Important, please fix

**Comment by OmegaWeaponDev** [comment]
This appears to have malicious code in it. I recommend closing and not merging for the safety of all.

File: src/data/editor-demo.json [hunk 1]
```diff
@@ -374,18 +374,6 @@
         }
       ]
     },
-    {
-      "type": "user",
-      "id": "b28021fd-9e1c-4a2b-af90-2df4535b702a",
-      "displayName": "Merjio",
```
Comment (Turbotailz) [inline src/data/editor-demo.json:380]: who tf

Comment (Aberdeener) [reply src/data/editor-demo.json:380]: thats why theyre gone

**Pull request merged**